app/views/games/show.html.erb

```diff
@@ -2,13 +2,15 @@
 document.addEventListener("DOMContentLoaded", function(){
   let activePiece;
   let destinationSquare;
-  let activePlayer = "white";
-  let inactivePlayer = "black";
+  let activePlayer = ( setActivePlayer = () => {
+    if (localStorage.getItem('ActivePlayer') == null) {
+      localStorage.setItem('ActivePlayer', 'white');
+    }
+    return localStorage.getItem('ActivePlayer');
+  })();
 
-  //Setup Promise to Grab board

```

Comment:
I believe these comments helped with readability - is this something we're going to rethink? Or just remove all comments?

Reply:
If you preferred them we can add them back in. It just felt like most of our names had enough intent behind them when I was going through, especially now that it is cleaned up.

Reply:
That works for me! I'll mark this as resolved